docs/how/auth/jaas.md

```diff
@@ -0,0 +1,71 @@
+# JaaS Authentication
+
+## Overview
+
+The DataHub frontend server comes with support for plugging in [JaaS](https://docs.oracle.com/javase/7/docs/technotes/guides/security/jaas/JAASRefGuide.html) modules. 
+This allows you to use a custom authentication protocol to log your users into DataHub. 
+
+By default, we in include sample configuration of a file-based username / password authentication module ([PropertyFileLoginModule](http://archive.eclipse.org/jetty/8.0.0.M3/apidocs/org/eclipse/jetty/plus/jaas/spi/PropertyFileLoginModule.html))
+that is configured with a single username / password combination: datahub - datahub. 
```

Comment:
And once we have fine grained access controls, this login module allows us to define the use role as well 👍

Reply:
It does!

Reply:
Heck yeah!